warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/TermOffsetPopulator.java

```diff
@@ -234,11 +193,48 @@ protected Range getRange(Set<Key> keys) {
         
         // Load the actual map into map that will be put into the JexlContext
         Map<String,Object> map = new HashMap<>();
-        map.put(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME, termOffsetMap);
-        
+        map.put(TERM_OFFSET_MAP_JEXL_VARIABLE_NAME, termOffsetMap);
         return map;
     }
     
+    /**
+     * Build a range from the search space.
+     *
+     * @param row
+     *            the shard
+     * @param searchSpace
+     *            a sorted set of TF column qualifiers
+     * @return a range encompassing the full search space
+     */
+    public static Range getRangeFromSearchSpace(Text row, TreeSet<Text> searchSpace) {
```

Comment:
Perhaps we can indeed use `SortedSet` instead of `TreeSet`?

Reply:
The searchSpace object is passed to the TermFrequencyOffsetIterator which makes use of the TreeSet's `higher(E)` method. It's convent to leave this as a TreeSet.